apps/commudle-admin/src/app/feature-modules/dashboard/components/user-dashboard/dashboard-updates/dashboard-updates.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, ElementRef, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { ICommunity } from '@commudle/shared-models';
+import {
+  faComments,
+  faPlus,
+  faLightbulb,
+  faTrophy,
+  faFileText,
+  faFlask,
+  faBullhorn,
+} from '@fortawesome/free-solid-svg-icons';
+import { ERegistrationStatuses } from 'apps/shared-models/enums/registration_statuses.enum';
+import { UsersService } from 'apps/shared-services/users.service';
+import moment from 'moment';
+// import { generate } from 'lean-qr';
+
+@Component({
+  selector: 'commudle-dashboard-updates',
+  templateUrl: './dashboard-updates.component.html',
+  styleUrls: ['./dashboard-updates.component.scss'],
+})
+export class DashboardUpdatesComponent implements OnInit {
+  @ViewChild('qrCanvas', { static: true }) qrCanvas: ElementRef<HTMLCanvasElement>;
+  @ViewChild('postContentBox') postContentBox: TemplateRef<any>;
+
+  ERegistrationStatuses = ERegistrationStatuses;
+  showEntryPass: boolean[] = [false];
+  faPlus = faPlus;
+  faTrophy = faTrophy;
+  faFileText = faFileText;
+  faFlask = faFlask;
+  faBullhorn = faBullhorn;
```

Comment:
[SUGGESTION] For cleaner code when can create a object named as icon and define icon name inside it!